core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaGameEventTranslator.java

```diff
@@ -51,52 +51,54 @@ public class JavaGameEventTranslator extends PacketTranslator<ClientboundGameEve
     @Override
     public void translate(GeyserSession session, ClientboundGameEventPacket packet) {
         PlayerEntity entity = session.getPlayerEntity();
+        // Strength of rainstorms and thunderstorms is a 0-1 float on Java, while on Bedrock it is a 0-65535 int 
+        final int MAX_STORM_STRENGTH = 65535;
```

Comment:
Just make this a private static constant.

Reply:
Like in Constants.java or just there?

Reply:
https://github.com/GeyserMC/Geyser/pull/3637/commits/5da9d2b67ee6191b09bb078b4d0de0b52e68519b